docs/development/dependencies.rst

```diff
@@ -0,0 +1,24 @@
+Upgrading HTMX to a new version
+===============================
+
+To upgrade the HTMX to a new version, follow these steps:
+
+1. **Backup Current File**:
+   Ensure you have a backup of the current `htmx-*.min.js` file located in the `static` directory.
+
+2. **Download the Latest Version**:
+   Obtain the latest version of minified HTMX from the `UNPKG`_, for example:
+   `https://unpkg.com/htmx.org@2.0.2/dist/htmx.min.js`
+
+3. **Replace Old File**:
+   Replace the old HTMX file in your project with the new file from the latest version. This involves updating the `htmx-*.min.js` file in the `static` directory. Make sure to replace the `*` in the file name with the new version number.
```

Comment:
Nitpick: this could make a great Make command

Reply:
The version-number wrangling would make it quite clunky.

Reply:
We could have a management command that does this, which could be run from tox/make.